### DISCUSSION TITLE
coqui-tts cache dir?

### DISCUSSION THREAD

**Discussion by ROBERT-MCDOWELL** [idiap/coqui-ai-TTS#194] [Q&A] [answered]
Is anyone know how we can move the cache dir to a dir we want to choose?
I tried all ENV from the docs and nothing changed. when I load xtt-v2 from the net
it always download it in the default TTS user cache dir.
thanks!

**Comment by eginhard** [comment, marked as answer]
> I tried all ENV from the docs and nothing changed. when I load xtt-v2 from the net

Please always be specific: what did you try, what did you expect, what actually happened? The following works fine for me:
```bash
export TTS_HOME=/path/to/download/dir
tts --text "hello" --model_name "tts_models/multilingual/multi-dataset/xtts_v2"
```

**Comment by ROBERT-MCDOWELL** [comment]
I mean from a python script, not command line. I tried all ways according to the docs, without success

**Reply by eginhard** [reply]
It's the same, you run `export TTS_HOME=/path/to/download/dir` and then your Python script. And again, instead of "I tried all ways according to the docs" please state exactly what you tried.

**Reply by ROBERT-MCDOWELL** [reply]
sorry I was asking as python like
os.environ['TTS_HOME'] = '/zzz/xxx/cache/dir'
this never worked for me :(

**Reply by ROBERT-MCDOWELL** [reply]
ok I just realized that I used HF_HOME until now. I'm going to test with TTS_HOME thanks

**Reply by ROBERT-MCDOWELL** [reply]
damn it works... thanks mate!
